integrationtest/pom.xml

```diff
@@ -160,7 +150,12 @@
                                         <groupId>org.hibernate</groupId>
                                         <artifactId>hibernate-search-modules</artifactId>
                                         <version>${hibernateSearchVersion}</version>
-                                        <classifier>${hibernateWildflyClassifier}</classifier>
+                                        <!--
+                                        We don't have a wildfly-11-dist version of the Hibernate Search 5.6 modules.
+                                        Using the ones built for WildFly 10 will do so we hardcode the classifier here
+                                        for now.
+                                        -->
+                                        <classifier>wildfly-10-dist</classifier>
```

Comment:
I'm a bit concerned that we might forget to change this in the future, could you use a separate property and add it close toHibernateSearchVersion or version.wildfly or hibernateWildflyClassifier, so that when we upgrade the libraries we remember to have a look at this? Does it make sense?

Reply:
We won't as I will rebase my ORM 5.2 PR right away and we won't have this weird thing there.

That's why I didn't bother about adding a property.

Reply:
ok